cpp/src/groupby/sort/aggregate.cpp

```diff
@@ -792,6 +808,24 @@ void aggregate_result_functor::operator()<aggregation::MERGE_TDIGEST>(aggregatio
                                                               mr));
 }
 
+template <>
+void aggregate_result_functor::operator()<aggregation::MERGE_HLLPP>(aggregation const& agg)
+{
+  if (cache.has_result(values, agg)) { return; }
+
+  int const precision =
+    dynamic_cast<cudf::detail::merge_hyper_log_log_aggregation const&>(agg).precision;
+
```

Comment:
```suggestion
  int const precision =
    dynamic_cast<cudf::detail::merge_hyper_log_log_aggregation const&>(agg).precision;
```

Reply:
Sorry, my eyes aren't able to pick up the difference suggested here.  What's the change?

Reply:
I suspect removal of the blank line, but github has a bug that doesn't show it.